earthpy/plot.py

```diff
@@ -407,7 +407,16 @@ def make_col_list(unique_vals, nclasses=None, cmap=None):
     -------
     >>> import numpy as np
     >>> import earthpy.plot as ep
-
+    >>> import matplotlib.pyplot as plt
+    >>> bins = [-np.inf, 2, 7, np.inf]
+    >>> im_arr_bin = np.digitize(np.random.randint(10, size=(15, 15)), bins)
+    >>> f, ax = plt.subplots()
+    >>> im = ax.imshow(im_arr_bin, cmap="Blues")
+    >>> the_legend = ep.draw_legend(im_ax=im)
+    # Get the array and cmap from axis object
```

Comment:
Please test `make docs` and `pytest` locally before submitting prs. docs won't build here because you've added a comment without ```>>>```